bpf/types.h

```diff
@@ -77,6 +78,7 @@ typedef struct flow_metrics_t {
     // 0 otherwise
     // https://chromium.googlesource.com/chromiumos/docs/+/master/constants/errnos.md
     u8 errno;
+    u8 dscp;
```

Comment:
@msherif1234 do you think with the growing structure we should turn this into a union of sub structures which cannot occur at the same time. 
Will save us space per entry of the hashmap as we grow to more protocols and types.

Reply:
Maybe one 8 bit dhcp field is not too big a change to say this, I should have mentioned it before when tcp drops and dns was added but just mentioning this now to open disucssion.